modules/phase_field/include/kernels/KKSSplitCHCRes.h

```diff
@@ -38,6 +38,7 @@ class KKSSplitCHCRes : public DerivativeMaterialInterface<SplitCHBase>
   virtual Real computeDFDC(PFFunctionType type);
   virtual Real computeQpOffDiagJacobian(unsigned int jvar);
   virtual Real computeQpResidual();
```

Comment:
The `computeQp...` methods should actually be protected. Would you mind changing that?